avocado/utils/multipath.py

```diff
@@ -115,6 +115,48 @@ def get_paths(wwid):
     return paths
 
 
+def get_multipath_details():
+    """
+    Get multipath details as a dictionary, as given by the command:
+    multipathd show maps json
+
+    :return: Dictionary of multipath output in json format.
+    """
+    mpath_op = process.system_output("multipathd show maps json", shell=True)
```

Comment:
Parsing traditional output is pretty difficult and does not make sense, when json format is available. But I will try to find out if multipathd version which does not support is old enough for us to assume the risk.